dev/source/docs/roadmap.rst

```diff
@@ -1,120 +1,118 @@
-
-Copter (Randy / Leonard / Tridge)
----------------------------------
-
-- ESC feedback handling
-- Flight Mode Improvements
-
-  - System identification mode
-  - 4kHz+ loop rate PIDs
-  - Stand-By mode
-
-- Attitude controller
-
-  - Separate FF and PID input for correct scaling
-  - Rate loop updates
-  - Enable rate loop saturation from external sources
-  - SI unit input
-
-- Position Controller
-
-  - Update Z to XY feed forward architecture
-  - Include position error limits based on velocity saturation
-  - Include velocity error limits based on accel saturation
-  - Handle EKF reset for position correctly
+Rover & Boat
+------------
 
-- AutoTune
+- S-Curve navigation (`video from 2019 un-conference <https://www.youtube.com/watch?v=LHq5o9zgNWk>`__) (Leonard/Randy)
+- Stick Mixing in Auto (TomP)
+- Underwater mapping with scanning sonar (Randy/PeterB)
+- Precision Docking (using marker onshore) (Randy)
+- high speed sailing inprovements (Randy)
+- Roll and pitch control for boats (Randy)
+- AIS (i.e., ADSB for boats) (Randy)
 
-  - Add tuning type to PID object
 
 Trad Heli (Bill Geyer, Chris Olson)
 -----------------------------------
 
-- Closed loop rotor speed governor for gas and turbine engine helicopters
-- Automatic engine failure identification and autorotation entry
-- Autonomous Autorotation
-- L1 navigation and speed/height controller for helicopter high speed autonomous missions
-- Improve ground handling and ground/air transitions
-- Improve rotor speed control library architecture
-- Tie motor spool states to measured rotor speed when measurement is available
-- Virtual flybar option for acro flight mode
+- Automated handling of engine throttle for autonomous operations (Bill Geyer, GSoC)
+- Automatic engine failure identification and autorotation entry (Matt Kear)
+- Autonomous Autorotation (Matt Kear)
+- Handling of manual autorotation (Matt Kear)
+- Heli Autotune (Bill Geyer)
+- Improve ground handling and ground/air transitions (Bill Geyer/Matt Kear)
+- Improve high-speed autonomous maneuvering (Bill Geyer, GSoC)
+- Improved shaping functions in the attitude controller (Bill Geyer, GSoC)
+- Tie motor spool states to measured rotor speed when a measurement is available (Bill Geyer)
 
-Rover & Boat
-------------
 
-- S-Curve navigation (`video from 2019 un-conference <https://www.youtube.com/watch?v=LHq5o9zgNWk>`__) (Leonard/Randy)
-- Stick Mixing in Auto (TomP)
-- Underwater mapping with scanning sonar (Randy/PeterB)
-- Precision Docking (using marker on shore) (Randy)
-- 3G/LTE telemetry (Randy)
-- APSync to support connecting to external Wifi access point (Randy/Peter)
+EKF (PaulR / Randy)
+-------------------
 
-Documentation (Randy / PierreK)
--------------------------------
+- Bring the missing features from EKF2 into EKF3 to remove EKF2 (Paul)
+- EKF3 External Nav support
+- Fixing replay (PeterB)
+- Kill EKF2 after feature capability of EKF3
+- Possibly switch to copter PID controllers
```

Comment:
"Possibly switch to copter PID controllers" should be moved to the plane section

Reply:
ok, done.